aten/src/ATen/native/native_functions.yaml

```diff
@@ -13686,6 +13686,13 @@
   tags: view_copy
   autogen: _test_autograd_multiple_dispatch_view_copy.out
 
+# Note: this function is only for testing in TestPythonDispatch.test_return_stream
```

Comment:
Ho you don't need a brand new native function for this, you can define it directly from python for testing:
```python
import torch
from torch.library import Library

l = Library("aten", "DEF")
l.define("_test_return_stream() -> Stream")

# If you need an implementation, you can register it as l.imp()
# You can then call this native op via torch.ops.aten._test_return_stream just like this one.

```